stdlib/src/csv.rs

```diff
@@ -32,21 +43,381 @@ mod _csv {
         )
     }
 
+    fn new_csv_error(vm: &VirtualMachine, msg: String) -> PyBaseExceptionRef {
+        vm.new_exception_msg(super::_csv::error(vm), msg)
+    }
+
+    #[pyattr]
+    #[pyclass(module = "csv", name = "Dialect")]
+    #[derive(Debug, PyPayload, Clone, Copy)]
+    struct PyDialect {
+        delimiter: u8,
+        quotechar: Option<u8>,
+        escapechar: Option<u8>,
+        doublequote: bool,
+        skipinitialspace: bool,
+        lineterminator: csv_core::Terminator,
+        quoting: QuoteStyle,
+        strict: bool,
+    }
+    impl Constructor for PyDialect {
+        type Args = PyObjectRef;
+
+        fn py_new(cls: PyTypeRef, ctx: Self::Args, vm: &VirtualMachine) -> PyResult {
+            PyDialect::try_from_object(vm, ctx)?
+                .into_ref_with_type(vm, cls)
+                .map(Into::into)
+        }
+    }
+    #[pyclass(with(Constructor))]
+    impl PyDialect {
+        #[pygetset]
+        fn delimiter(&self, vm: &VirtualMachine) -> PyRef<PyStr> {
+            vm.ctx
+                .intern_str(format!("{}", self.delimiter as char))
+                .to_owned()
+        }
+        #[pygetset]
+        fn quotechar(&self, vm: &VirtualMachine) -> PyObjectRef {
+            match self.quotechar {
+                Some(t) => vm
+                    .ctx
+                    .intern_str(format!("{}", t as char))
+                    .to_owned()
+                    .into(),
+                None => vm.ctx.none(),
+            }
+        }
+        #[pygetset]
+        fn doublequote(&self, vm: &VirtualMachine) -> PyRef<PyInt> {
+            vm.ctx.new_bool(self.doublequote).to_owned()
+        }
+        #[pygetset]
+        fn skipinitialspace(&self, vm: &VirtualMachine) -> PyRef<PyInt> {
+            vm.ctx.new_bool(self.skipinitialspace).to_owned()
+        }
+        #[pygetset]
+        fn lineterminator(&self, vm: &VirtualMachine) -> PyRef<PyStr> {
+            match self.lineterminator {
+                Terminator::CRLF => vm.ctx.intern_str("\r\n".to_string()).to_owned(),
+                Terminator::Any(t) => vm.ctx.intern_str(format!("{}", t as char)).to_owned(),
```

Comment:
is this `t` static value or dynamic value?

`intern_str` [interns](https://en.wikipedia.org/wiki/String_interning) static strings. If this is a dynamic value, 

```suggestion
                Terminator::Any(t) => vm.ctx.new_str(format!("{}", t as char)).to_owned(),
```

Reply:
Haha, I didn't notice this API when I was looking through the API documentation. I was still wondering why there wasn't a basic API for creating dynamic strings.